jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -151,6 +151,7 @@ public static synchronized void init() {
             if (DRIVER.name.equals(DEFAULT_DRIVER))
                 fillAction(p -> DRIVER.name = p, "driver");
             fillAction(p -> DRIVER.version = p, "driver.version");
+            fillAction(p -> DRIVER.type = p, "driver.type");
```

Comment:
change to parse string value with ignore case

Reply:
Done